libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/StatsTableUtils.ts

```diff
@@ -122,23 +123,28 @@ export function generateCohortsStatsTable(
           });
         } else {
           // not a numeric value (NaN), so just put null and use textured color
-          items.push({
-            color: {
-              pattern: {
-                aspectRatio: 1,
-                backgroundColor: "white",
-                color: "pink",
-                height: 10,
-                image: "",
-                opacity: 0.5,
-                path: {
-                  d: "M 0 0 L 10 10 M 9 -1 L 11 1 M -1 9 L 1 11",
-                  strokeWidth: 3
-                },
-                patternTransform: "",
-                width: 10
+          const colorConfig = useTexturedBackgroundForNaN
+            ? {
+                color: {
+                  pattern: {
+                    aspectRatio: 1,

```

Comment:
hmm can this be moved into the styles file?

Reply:
It's not an `IStyle` but I can try.

Reply:
No, doesn't work since it's not expecting an IStyle. Will leave it as is.